src/main/scala/mesosphere/marathon/plugin/PluginModule.scala

```diff
@@ -0,0 +1,30 @@
+package mesosphere.marathon.plugin
+
+import java.net.URL
+
+import com.google.inject.{ Singleton, Provides, AbstractModule }
+import org.rogach.scallop.ScallopConf
+
+trait PluginConfiguration extends ScallopConf {
+  lazy val pluginUris = opt[List[String]]("plugin_uris",
```

Comment:
URIs are powerful, but also complicated. My proposal would be to use plugin_dir. That directory is listed and all items are added to the CP.
Example: `/path/to/marathon/plugins`

Perhaps it make sense to use that option multiple times.